glom/core.py

```diff
@@ -1133,6 +1133,57 @@ def glomit(self, target, scope):
         return ret
 
 
+class Capture(object):
```

Comment:
capture is really its own thing right? doesn't seem to directly influence match?

Reply:
yes this could (should?) be broken into it's own thing

Reply:
capture is in the area of pattern matching, basically it is adjacent to the match concept so I was kind of prototyping it in the same place